specs/kubernetes/list_pod_images_by_app.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: List images by app label of pods
```

Comment:
nit: could we make the name of this workflow more specific to the tool?

How about: 
"List all Kubernetes pods by app label"?

Reply:
Changed according to your suggestion.